src/Package.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Returns information about the package and handles init.
+ *
+ * @package Automattic/WooCommerce/WCAdmin
+ */
+
+namespace Automattic\WooCommerce\Admin;
+
+defined( 'ABSPATH' ) || exit;
+
+require_once WC_ABSPATH . 'packages/woocommerce-admin/includes/core-functions.php';
```

Comment:
I see you're using `require_once` here which is good, but I see you had to add `function_exists` to all of the functions included in this files. Why was that?

Another alternative would be to move these requires statements to your `FeaturePlugin` class (which is autoloaded once) to include during `init`. Maybe that will remove the need for function exists?

Reply:
Why was that?

There were conflicts, but your suggestion worked, thanks!